docs/_docs/installation/macos.md

```diff
@@ -3,155 +3,28 @@ title: Jekyll on macOS
 permalink: /docs/installation/macos/
 ---
 
-## Install Command Line Tools
-To install the command line tools to compile native extensions, open a terminal and run:
+## Supported macOS versions
 
-```sh
-xcode-select --install
-```
+- Monterey (macOS 12)
+- Big Sur (macOS 11)
+- Catalina (macOS 10.15)
 
-### set SDKROOT (only macOS Catalina or later)
-Starting on macOS Catalina (10.15) the headers used for Ruby have been moved from their previous location which results in some gems, including Jekyll to fail installation. This can be solved by setting SDKROOT in your shell configuration to the value provided by xcrun.
-
-```ssh
-export SDKROOT=$(xcrun --show-sdk-path)
-```
+Older macOS versions might work, but we don't officially support them.
 
 ## Install Ruby
 
-Jekyll requires **Ruby v{{ site.data.ruby.min_version }}** or higher.
-macOS Big Sur 11.x ships with Ruby 2.6.3. Check your Ruby version using `ruby -v`.
-
-If you're running a previous version of macOS, you'll have to install a newer version of Ruby. Installation with [Homebrew](https://brew.sh) is simple if you're only planning to use Ruby for Jekyll. Install with a version manager such as [asdf](https://asdf-vm.com/), [chruby](https://github.com/postmodern/chruby), [rbenv](https://github.com/rbenv/rbenv), or [rvm](https://rvm.io/) if you need to switch among Ruby versions (instructions for rbenv are below). See the guide [Install Ruby on Mac](https://mac.install.guide/ruby/index.html) for details and recommendations.
-
-### With Homebrew {#brew}
```

Comment:
While Homebrew is an option for installing Ruby, I don't recommend mentioning it because eventually it will cause people confusion and headaches when they need to work with different projects that use different versions of Ruby.

Given that installing Ruby with a version manager is just as easy, and it has the additional benefit of supporting multiple versions at the same time, we should favor it. People prefer to be given clear instructions that will serve them in the long term, as opposed to making them choose between various options that they don't fully understand.

Furthermore, the Homebrew gems path that needs to be added to the `PATH` depends on the version of Ruby that was installed with Homebrew, which means these instructions below only work if someone installed Ruby 3.0.x. Sure, we could update these instructions to say to replace `3.0` with the first 2 digits of the Ruby version, but why make things more complicated than necessary?